src/libs/mysqlHandler.go

```diff
@@ -79,12 +79,12 @@ func GetNetworkPoliciesFromMySQL(cfg types.ConfigDB, cluster, namespace, status,
 	}
 
 	results, err = DBHandle.Query(query+whereClause, args...)
-	defer results.Close()
 
 	if err != nil {
 		log.Error().Msg(err.Error())
 		return nil, err
 	}
+	defer results.Close()
```

Comment:
This change seems to be incorrect. The `Close()` should be done only if the `Query()` is successful. The `results.Close()` should not be done if the `err != nil`.

Reply:
Had the same confusion, but moved this handle as per existing db functions. Will handle this.

Reply:
![Screenshot from 2022-07-25 16-35-04](https://user-images.githubusercontent.com/89014588/180763334-12ea5a0a-db49-4327-9dc1-cef8d3b43a5a.png)

Reply:
Suggestion is to check error before deferring results.Close() or stmt.Close()